packages/api/src/scheduler/validity-check.scheduler.ts

```diff
@@ -38,11 +40,11 @@ export const validityCheckScheduler = {
       () => {
         const now = new Date();
         return validityCheckService
-          .deleteByFinAPTBefore(now.getTime())
+          .deleteByFinAPTBefore(now)
           .pipe(
-            tap((res: DeletedData[]) =>
+            tap((res: Number) =>
```

Comment:
eslint/tslint prefer 
```suggestion
            tap((res: number) =>
```